scripts/getDependencies.pl

```diff
@@ -322,7 +321,7 @@
 		my $dir = $jars_info[$i]{dir} // "";
 		my $full_dir_path = File::Spec->catdir($path, $dir);
 		my $url_custom = $customUrl;
-
+		my $thirdParty_Url = $url;
```

Comment:
You can use camel case or `_`, but do not mix them. In this case, call it `third_party_url`